pangeo_forge_recipes/recipes/xarray_zarr.py

```diff
@@ -592,6 +592,20 @@ def store_chunk(
             var_coded.attrs = {}
             with dask.config.set(scheduler="single-threaded"):  # make sure we don't use a scheduler
                 var = xr.backends.zarr.encode_zarr_variable(var_coded)
+                logger.debug(
+                    f"Converting variable {vname} of {var.data.nbytes} bytes to `numpy.ndarray`"
+                )
+                if var.data.nbytes > 500_000_000:
+                    oversize_factor = round((var.data.nbytes/(500_000_000)), 2)
+                    logger.warning(
+                        f"Variable {vname} of {var.data.nbytes} bytes is {oversize_factor} times"
+                        " larger than recommended maximum variable array size of 500 MB. "
+                        ' Consider re-instantiating recipe with `subset_inputs = {"time": '
```

Comment:
Let's not assume `time` is the name of the `concat_dim`. We have access to the real `concat_dim` here!

```suggestion
                        ' Consider re-instantiating recipe with `subset_inputs = {"{concat_dim}": '
```